server/graphql/v2/collection/MemberCollection.js

```diff
@@ -27,6 +28,9 @@ export const MemberOfCollection = new GraphQLObjectType({
       nodes: {
         type: new GraphQLList(MemberOf),
       },
+      roles: {
+        type: new GraphQLList(GraphQLJSON),
```

Comment:
Why JSON rather than a strong type?

Reply:
I wasn't sure if I should present the collective type in the same array at the beginning so I left this as a prototype.
Now that I know we require roles per collective type I can add a proper type to this.